cpp/include/cuspatial/distance.hpp

```diff
@@ -127,118 +127,64 @@ std::pair<std::unique_ptr<cudf::column>, cudf::table_view> directed_hausdorff_di
   rmm::mr::device_memory_resource* mr = rmm::mr::get_current_device_resource());
 
 /**
- * @brief Compute pairwise (multi)point-to-(multi)point Cartesian distance
+ * @brief Compute pairwise (multi)point-to-(multi)point Euclidean distance
```

Comment:
Why Euclidean and not Cartesian? AFAIU Cartesian is Euclidean + a coordinate system.

Reply:
Thanks. I accept that "Cartesian distance" is short for Euclidean distance on Cartesian CRS. The reason why I hesitated is that there are other distance definition on a cartesian plane, such as Manhatten distance. But since Euclidean is the most commonly used distance (at least in GIS terms), it should be safe to assume that.

Reply:
Got it. Good point.